src/installer/corehost/cli/test/mockhostfxr/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+cmake_minimum_required (VERSION 2.6)
+project(mockhostfxr)
```

Comment:
Since the test relies on the fact that the hostfxr will not have custom error writer entry point, I think it would be cleaner to call this `mockhostfxr_2_2`. Basically make it obvious that this is trying to imitate the pre-3.0 hostfxr.
I can imagine that one day we'll have a 3.0 hostfxr mock, which will have the error writer.